core-web/libs/data-access/src/lib/dot-site/dot-site.service.spec.ts

```diff
@@ -1,69 +1,54 @@
-import { HttpClientTestingModule, HttpTestingController } from '@angular/common/http/testing';
-import { TestBed } from '@angular/core/testing';
+import { createHttpFactory, HttpMethod, SpectatorHttp } from '@ngneat/spectator/jest';
 
 import { mockSites } from '@dotcms/utils-testing';
 
 import { DotSiteService, SiteParams } from './dot-site.service';
 
 describe('DotSiteService', () => {
+    let spectator: SpectatorHttp<DotSiteService>;
     let service: DotSiteService;
-    let httpTestingController: HttpTestingController;
+
+    const createHttp = createHttpFactory(DotSiteService);
 
     const BASE_SITE_URL = '/api/v1/site';
```

Comment:
Centralize this constant, and reuse it in the service and test.

Reply:
@oidacra done!